smartsim/_core/_install/buildenv.py

```diff
@@ -189,17 +189,6 @@ class RedisAIVersion(Version_):
     """
 
     defaults = {
-        "1.2.5": {
-            "tensorflow": "2.6.2",
-            "onnx": "1.9.0",
-            "skl2onnx": "1.10.3",
-            "onnxmltools": "1.10.0",
-            "scikit-learn": "1.0.2",
-            "torch": "1.9.1",
-            "torch_cpu_suffix": "+cpu",
-            "torch_cuda_suffix": "+cu111",
-            "torchvision": "0.10.1",
-        },
```

Comment:
You have no idea how happy this makes me! Goodbye TF 2.6.x!! 🎉

Reply:
Progress is unstoppable!